src/main.cpp

```diff
@@ -495,6 +508,22 @@ int main(int argc, char** argv) try {
         throw std::runtime_error("ORC could not find files to process");
     }
 
+    // Count the ODRVs. FIXME See `filter_print_report` for code that needs refactoring.
+    // The stream_to_nowhere is a temporary work-around.
+    for (const auto& report : orc_process(file_list)) {
+        std::stringstream stream_to_nowhere;
```

Comment:
If performance becomes a problem here, we should make a "null stream" type that receives whatever it's given for "output", but does nothing with it.

Reply:
all this re-written.

Reply:
wasn't sure what I was thinking :)